databases/week1/database.sql

```diff
@@ -0,0 +1,44 @@
+SELECT COUNT(*) AS total_tasks FROM task;
+
+SELECT COUNT(*) AS invalid_due_dates 
+FROM task 
+WHERE due_date IS NULL 
+
+SELECT * 
+FROM task 
+WHERE status_id = (SELECT id FROM status WHERE name = 'Done');
+
+SELECT * 
+FROM task 
+WHERE status_id != (SELECT id FROM status WHERE name = 'Done');
+
+SELECT * 
+FROM task 
+ORDER BY created DESC;
+
+SELECT * 
+FROM task 
+ORDER BY created DESC 
+LIMIT 1;
+
+SELECT title, due_date
+FROM task
+WHERE title LIKE '%database%' 
+OR description LIKE '%database%';
+
+SELECT task.title, status.name AS status
+FROM task
+JOIN status ON task.status_id = status.id;
+
+SELECT status.name, COUNT(task.id) AS task_count
+FROM status
+LEFT JOIN task ON status.id = task.status_id
+GROUP BY status.name;
```

Comment:
Amazing! You had the attention to detail to use `LEFT JOIN` here, instead of a simple `JOIN`. This is the correct way to do it, since this means that if a status does not have a task that would be associated with it, the status would still show up in the result with a `task_count` of 0. With a simple `JOIN` this would not happen. Well done!!! 🤩